inductiva/utils/meta.py

```diff
@@ -59,3 +60,32 @@ def get_method_name(function_ptr) -> str:
     module_name = ".".join(module_name_useful_segments)
 
     return f"{module_name}.{function_ptr.__name__}"
+
+
+# pylint: disable=unused-argument
+def custom_format_warning(msg, *args, **kwargs):
+    """Custom format for warnings."""
+    return str(msg) + "\n"
```

Comment:
why is this necessary?

Reply:
Without this the warning shows like this:
```
/Users/paulobarbosa/Documents/Gits/inductiva/inductiva/utils/meta.py:83): UserWarning: n_cores is deprecated, use n_vcpus instead.
  warnings.warn(f"{decorator_key} is deprecated, use "
```

Since meta.py is the code of the decorator that is meaningless to the user.